data/sudeepjhs.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Sudeep Agarwal",
+  "bio" : "Innovative Software Developer | JavaScript, ReactJS, Node.js, NextJS Enthusiast | Full Stack Developer | Outsystems Learner"
```

```suggestion
  "bio" : "Innovative Software Developer | JavaScript, ReactJS, Node.js, NextJS Enthusiast | Full Stack Developer | Outsystems Learner",
```
There is a comma missing

---

ok noted